apps/api-gateway/routers/helper.go

```diff
@@ -56,6 +56,16 @@ func parseResponse(route config.Route, respBody []byte) (interface{}, error) {
 	return resp, nil
 }
 
+func appendQueryParams(queries map[string]string, endpoint *string) {
+	if len(queries) >= 1 {
+		*endpoint += "?"
+	}
+
+	for key, value := range queries {
+		*endpoint += fmt.Sprintf("%s=%s&", key, value)
+	}
```

Comment:
Will also append on the last pair. Shouldn't really matter though.

Reply:
Yep, saw this coming but I think it's not worth the effort to cover this edge case